.github/workflows/gradle.yml

```diff
@@ -0,0 +1,36 @@
+name: Gradle Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
```

Comment:
Could we have this name be more descriptive, something like `gradle-build-and-test`

Reply:
That makes sense, thanks!